packages/storybook/src/stories/1-MarkdownEditor.stories.js

```diff
@@ -106,7 +106,7 @@ export const Demo = () => {
         children: slateChildren
       }
     };
-    const markdown = slateTransformer.toMarkdown(slateValue);
+    const markdown = slateTransformer.toMarkdownCicero(slateValue);
```

Comment:
Although here we shouldn't need this change, should we? Since the Markdown Editor does not handle cicero elements, only the Contract Editor does.

Reply:
I missed that. Agreeing with @DianaLease 's assessment.

Reply:
It should remain consistent with: https://github.com/accordproject/web-components/blob/00ee8b97b63d9003443642104402cf704d4b3c8d/packages/storybook/src/stories/1-MarkdownEditor.stories.js#L86

Reply:
Agreed. Overzealous find/replace...